src/ShapeCatalog.js

```diff
@@ -0,0 +1,150 @@
+    return(
+      <div className={`mdc-shape-container__corner mdc-shape-container__corner--${position}`}></div>
+    );
+  }
+
+  render() {
+    const initRipple = el => el && this.ripples.push(new MDCRipple(el));
+    return (
+      <div>
+        <h3>Contained Button</h3>
+        <div className='contained-button-shape-container mdc-shape-container'>
+          <button className='mdc-button mdc-button--unelevated' ref={initRipple}>Skip</button>
+          {this.renderCorner('top-left')}
+          {this.renderCorner('bottom-right')}
+        </div>
+        <div className='contained-button-shape-container mdc-shape-container'>
+          <button className='mdc-button mdc-button--unelevated' ref={initRipple}>Finish</button>
+          {this.renderCorner('top-left')}
+          {this.renderCorner('top-right')}
+          {this.renderCorner('bottom-left')}
+          {this.renderCorner('bottom-right')}
+        </div>
+
+        <h3>Outlined Button</h3>
+        <div className='outlined-button-shape-container mdc-shape-container'>
+          <button className='mdc-button mdc-button--outlined' ref={initRipple}>Skip</button>
+          {this.renderCorner('top-left')}
+          {this.renderCorner('bottom-right')}
+        </div>
+        <div className='outlined-button-shape-container mdc-shape-container'>
+          <button className='mdc-button mdc-button--outlined' ref={initRipple}>Finish</button>
+          {this.renderCorner('top-left')}
+          {this.renderCorner('top-right')}
+          {this.renderCorner('bottom-left')}
+          {this.renderCorner('bottom-right')}
+        </div>
+
+        <h3>Card</h3>
+        <div className='card-shape-container mdc-shape-container'>
+          <div class='mdc-card mdc-card--outlined'>
+            <div class='mdc-card__primary-action' ref={initRipple}>
+              <div class='demo-card__primary'>
+                <h2 class='demo-card__title mdc-typography--headline6'>Our Changing Planet</h2>
+                <h3 class='demo-card__subtitle mdc-typography--subtitle2'>by Kurt Wagner</h3>
+              </div>
+              <div class='demo-card__secondary mdc-typography--body2'>
+                Visit ten places on our planet that are undergoing the biggest changes today.
+              </div>
+            </div>
+            <div class='mdc-card__actions'>
+              <div class='mdc-card__action-icons'>
+                <i class='mdc-icon-toggle material-icons mdc-card__action mdc-card__action--icon'
+                  tabindex='0'
+                  role='button'
+                  aria-pressed='false'
+                  aria-label='Add to favorites'
+                  title='Add to favorites'
+                  data-toggle-on='{"content": "favorite", "label": "Remove from favorites"}'
+                  data-toggle-off='{"content": "favorite_border", "label": "Add to favorites"}'
+                  ref={(surfaceEl) => this.iconToggle = surfaceEl && new MDCIconToggle(surfaceEl)}>
```

Comment:
Is this.iconToggle being used anywhere? Maybe should be used to destroy it in unmount?